mirascope/core/base/_utils/_setup_call.py

```diff
@@ -51,12 +51,11 @@ def setup_call(
     if not messages:
         prompt_template = get_prompt_template(fn)
         assert prompt_template is not None, "The function must have a docstring."
```

Comment:
nit: i just noticed this should say "The function must have a prompt template" not "docstring"

please make sure this is updated anywhere else we have this issue